docs/client-authorization.md

```diff
@@ -0,0 +1,26 @@
+---
+id: client-authorization
+title: NKN Client Authorization
+---
+
+When a NKN client joins NKN network, it sends a RPC request to seed list to inquire which node that it should connect by webscoket.
+
+After getting the websocket node information, the client connects to the node, and sends its client Id, public key to the node.
+
+Then the node needs to check that the client is authorized to use this **client address** . This is what client authorization is for.
+
+The main steps are here:
+1. A client connects to a node by webscoket;
+2. When getting a new connection from a client, the node generates 256 bit data as a **challenge** , and send it to the client. This action is *authChallenge*;

```

Comment:
might be good to say it's random data in 2 and 3

Reply:
Yes. fixed.